help/de/Help_ch_GUIMenus.xml

```diff
@@ -1143,11 +1161,11 @@
                 </entry>
 
                 <entry namest="c3">
-                  <para>Erstellt eine neue bankinterne Umbuchung per Online Banking.
+                  <para>Erstellt eine bankinterne Umbuchung per Online Banking.
                   </para>
                 </entry>
               </row>
-
+<!-- FIXME source: this menuitem is obsolet -->
```

Comment:
The other direction: enable it in the program again.

Just in case you want to hack  again. ;-)

Reply:
Du kennst dich doch besser aus damit. 

@cstim Wurde der vielleicht aufgrund eines Denkfehler "Alles außer SEPA raus!" entfernt? Oder ist da wirklich was kaputt gegangen?